server/api/user/index.js

```diff
@@ -1,7 +1,30 @@
 import express from 'express';
-
+import controller from './user.controller';
 const router = express.Router();
+// TODO: Verify user-id with token
+
+// GET => all user expenses for selected pool
+// Cannot GET /api/users/5/expense-pools-users/4/expenses
+router.get('/:id/expense-pools-users/:exp-pool-id/expenses',
```

Comment:
I think the the route we would want would be along the lines of:

```
api/users/:id/expense-pools/:expensePoolId/expenses
```
or possibly
```
api/users/:userId/expense-pools/:expensePoolId/expenses
```
so for the `.get` the first param would be:

```
'/:id/expense-pools/:expensePoolId/expenses`
```

That way you can reference them in the controller as `req.params.expensePoolId`

The params should be camelCase but the actual resource names in the url should be pluralized and hyphen separated. The docs are usually a good place to look to see the convention to use for variable names.

http://expressjs.com/en/guide/routing.html#route-parameters